### PR TITLE
Portfolio Milestone 2: Works Section

### PR DESCRIPTION
**Pull request by eroiyo**
Add Works Section

--The work section includes a lot of templates of projects
--More fogs for the background
--More height

Add Projects Template
--Project Templates features
    *Image of the project
    *Description of the project
    *Button to display more information (coming soon)
    *Tags

### CODE REVIEW

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @eroiyo,

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Please check the comments below for detailed feedback.
- [ ] Kindly test your project with the dev tools and make sure everything positioned properly 


## Optional Suggestions 💡
_The following point is highly recommended to boost your productivity , Please consider it 

- take advantage of the dev tools and make sure everything works properly on Samsung and iPhone devices.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: style.css [hunk 1]
```diff
@@ -180,3 +250,79 @@ li a {
...
+
+span {
+  color: white;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.button {
+  position: absolute;
+  left: 0;
```
[the remark this answers is not in the document]
Comment (Shaher-11) [reply style.css:288]: If you are using chrome then right click on the page and choose  inspect then click on the devise icon,
this way you will be able to test on all devices or at least the common ones

[the remark this answers is not in the document]
Comment (eroiyo) [reply style.css:288]: Thanks you.

**Comment by eroiyo** [comment]
Ok, prepared for other review!

Increased the spaces between name and title (the h1 and the h2)
Increased the spaces between project image and name
now the buttons use flex instead of absolute and now positioned properly
the work title are now 2 lines instead of one

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @eroiyo😃, 

Great work on making the changes requested by a previous reviewer 👏
Extra credit for the Live Demo Link 💯 Keep up the great work 👍 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] **OPTIONAL**
Kindly check the MIT link in your README because it's broken.
Please keep in mind that such a small mistake might leave a bad impression on future viewers. So, I highly recommend fixing it.

- [ ] **OPTIONAL**
Please correct the misspelling in the PR headline "Portafolio". Instead, it's "Portfolio".
I recommend installing [Grammarly](https://app.grammarly.com/) to check for such mistakes while typing.

_Check the **OPTIONAL** comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eroiyo** [comment]
Sure, it's time for other review, I did some changes!

--Circles and fogs CSS effects are now gone, now we use 2 different images for the background
--Projects cards are now properly saved wrapped with list tags
--Bottom button is now properly displayed
--Font and typographic have been established, and a lot of redundant code was deleted

Thanks for the review

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @eroiyo, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**